projects/angular/src/forms/styles/_checkbox.clarity.scss

```diff
@@ -128,7 +128,6 @@
   }
 
   .clr-form-compact .clr-checkbox-wrapper {
-    @include forms-mixins.form-flatten-compact-adjustments();
```

Comment:
Creating inconsistency between other form controls and radio / checkbox because of specificity being higher here and overriding what is in _containers.clarity.scss